metaflow/plugins/cards/ui/src/components/bar-chart.svelte

```diff
@@ -17,11 +17,11 @@
     BarController,
     LinearScale,
     CategoryScale,
-    PointElement
+    PointElement,
   );
 
   export let componentData: types.BarChartComponent;
-  const { config, data, labels } = componentData;
+  $: ({ config, data, labels } = componentData);
```

Comment:
Fun sidenote, svelte just announced they're changing this api for the same confusion we got caught up in here. Glad its fixed for now.